### PR TITLE
fix(OnyxStepper): Remove default model value and placeholder for onyx stepper

### PR DESCRIPTION
**Pull request by MajaZarkova**
Relates to #1608 

Remove default model value and placeholder for onyx stepper

## Checklist

- [x] If a new component is added, at least one [Playwright screenshot test](https://github.com/SchwarzIT/onyx/actions/workflows/playwright-screenshots.yml) is added
- [x] A changeset is added with `npx changeset add` if your changes should be released as npm package (because they affect the library usage)

### CODE REVIEW

File: packages/sit-onyx/src/components/OnyxStepper/OnyxStepper.vue [hunk 1]
```diff
@@ -27,7 +25,7 @@ const inputRef = ref<HTMLInputElement>();
 
 const emit = defineEmits<{
   /** Emitted when the input value changes. */
-  "update:modelValue": [value: number];
+  "update:modelValue": [value: number | undefined];
```
Comment (larsrickert) [inline packages/sit-onyx/src/components/OnyxStepper/OnyxStepper.vue:28]:
```suggestion
  "update:modelValue": [value?: number];
```

**Review by larsrickert — CHANGES_REQUESTED** [review]

**Review by BoppLi — APPROVED** [review]

File: packages/sit-onyx/playwright/snapshots/components/OnyxStepper/Stepper-labelTooltip-messageTooltip--webkit-linux.png
Comment (JoCa96) [inline packages/sit-onyx/playwright/snapshots/components/OnyxStepper/Stepper-labelTooltip-messageTooltip--webkit-linux.png:1]: Without a default value, it looks too empty. Can we keep `0` as a default value?

**Review by JoCa96 — CHANGES_REQUESTED** [review]

**Review by larsrickert — APPROVED** [review]

**Comment by JoCa96** [comment]
As discussed, please only emit in the `computed`. In the `handleClick` check for NaN and emit `undefined`.

**Review by JoCa96 — CHANGES_REQUESTED** [review]
As discussed, please only emit in the `computed`. In the `handleClick` check for NaN and emit `undefined`.

**Review by JoCa96 — APPROVED** [review]

**Pull request merged**